katas/content/teleportation/entangled_pair/Verification.qs

```diff
@@ -0,0 +1,28 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Katas;
+
+    operation Entangle_Reference (qAlice : Qubit, qBob : Qubit) : Unit is Adj {
+        H(qAlice);
+        CNOT(qAlice,qBob);
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        use qAlice = Qubit();
```

Comment:
Let's use the standardized method `CheckOperationsEquivalenceOnZeroStateWithFeedback` from KatasLibrary to make this check - we'll want to use the same feedback text and display of expected/actual state we're using in the rest of the katas, and this makes it easier. You'll need to define a wrapper for the Kata.Entangle solution to make it act on an array of qubits rather than on two individual qubits, but you can define Entangle_Reference to act on an array right away

Reply:
(You can see the updated code for Superdense Coding kata and how it does this)

Reply:
Changed using Superdense Coding kata